Source/UI/NewUI/PaymentForm/ViewController/PaymentViewController.swift

```diff
@@ -103,7 +104,12 @@ final class PaymentViewController: UIViewController {
   }
 
   private func setupNavigationBar() {
-    customizeNavigationBarAppearance(color: viewModel.paymentFormStyle?.headerView.backgroundColor ?? .white)
+    let backgroundColor = viewModel.paymentFormStyle?.headerView.backgroundColor ?? .white
+    let titleColor = viewModel.paymentFormStyle?.headerView.headerLabel?.textColor ?? .black
+
+    let titleFont = viewModel.paymentFormStyle?.headerView.headerLabel?.font.withSize(15) ?? UIFont.systemFont(ofSize: 15)
```

Comment:
Should we hardcode the size? May allow consumer to set it themselves and own size?

```suggestion
    let titleFont = viewModel.paymentFormStyle?.headerView.headerLabel?.font ?? UIFont.systemFont(ofSize: Constants.defaultNavigationHeaderFontSize)
```